docs/building/windows-core.md

```diff
@@ -77,3 +77,13 @@ You can run our cross-platform Pester tests with `Start-PSPester`.
 ## Building in Visual Studio
 
 We currently have the issue [#3400](https://github.com/PowerShell/PowerShell/issues/3400) tracking this task.
+
+## Building an installer
```

Comment:
Instead of calling it `Building an installer`, I think it's more correct to say `Building an Installable Package` (where installer is msi)